community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/PackStreamMessageFormatV1.java

```diff
@@ -136,6 +136,20 @@ public void handleRecordMessage( Record item )
                 packer.pack( field );
             }
             onMessageComplete.onMessageComplete();
+
+            //The record might contain unpackable values,
+            //hence we must consume any errors that might
+            //have occurred.
+            packer.consumeError( ( status, s ) -> {
```

Comment:
If I understand correctly, for each record we will only save the last error if there are more than one error while packing that records? Will it be better if we could save the errors as a error list? It would be interesting to see how it behavies in a test